src/screens/ProfileScreen/index.js

```diff
@@ -761,44 +759,44 @@ const styles = StyleSheet.create({
   seeMore: {
     fontFamily: fonts.inter[500],
     fontSize: 14,
-    color: colors.black
+    color: COLORS.black
   },
   tabs: {
     width,
-    borderBottomColor: colors.alto,
+    borderBottomColor: COLORS.alto,
     borderBottomWidth: 1,
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'center',
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
   },
-  tabItem: (isActive) => ({
+  tabItem: (isActive, isSigned) => ({
     flex: 1,
     alignItems: 'center',
     justifyContent: 'center',
     height: 48,
-    borderBottomColor: colors.bondi_blue,
+    borderBottomColor: isSigned ? COLORS.blue : COLORS.holyTosca,
     borderBottomWidth: isActive ? 2 : 0
   }),
   tabsFixed: {
     width,
-    borderBottomColor: colors.alto,
+    borderBottomColor: COLORS.alto,
     borderBottomWidth: 1,
     paddingLeft: 20,
     paddingRight: 20,
     flexDirection: 'row',
     position: 'absolute',
     top: 42,
     zIndex: 2000,
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
   },
   nameProfile: {
     fontFamily: fonts.inter[800],
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 17,
     marginTop: 12,
-    color: colors.black
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in color constants from `colors` to `COLORS` are consistent with the import statement change at the top of the file. This ensures that the correct color values are used throughout the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->